tests/ci/azureml_tests/submit_groupwise_azureml_pytest.py

```diff
@@ -14,492 +14,166 @@
         default="azuremlcompute",
-        help="Set name of Azure cluster",
+        help="AzureML cluster name",
     )
-    # Azure VM size
     parser.add_argument(
         "--vmsize",
         action="store",
         default="STANDARD_D3_V2",
-        help="Set the size of the VM either STANDARD_D3_V2",
+        help="VM size",
     )
-    # Azure subscription id, when used in a pipeline, it is stored in keyvault
     parser.add_argument(
-        "--subid", action="store", default="123456", help="Azure Subscription ID"
+        "--subid",
+        action="store",
+        default="123456",
+        help="Azure Subscription ID",
     )
-    # AzureML experiment name
     parser.add_argument(
         "--expname",
         action="store",
         default="persistentAzureML",
-        help="experiment name on Azure",
+        help="Experiment name on AzureML",
     )
-    # Azure datacenter location
-    parser.add_argument("--location", default="EastUS", help="Azure location")
-    # github repo, stored in AzureML experiment for info purposes
     parser.add_argument(
-        "--reponame",
+        "--envname",
         action="store",
-        default="--reponame MyGithubRepo",
-        help="GitHub repo being tested",
+        default="recommenders",
+        help="Environment name on AzureML",
     )
-    # github branch, stored in AzureML experiment for info purposes
-    parser.add_argument(
-        "--branch",
-        action="store",
-        default="--branch MyGithubBranch",
-        help=" Identify the branch test test is run on",
-    )
-    # github pull request, stored in AzureML experiment for info purposes
-    parser.add_argument(
-        "--pr",
-        action="store",
-        default="--pr PRTestRun",
-        help="If a pr triggered the test, list it here",
-    )
-    # flag to indicate whether gpu dependencies should be included in conda env
-    parser.add_argument(
-        "--add_gpu_dependencies",
-        action="store_true",
-        help="include packages for GPU support",
-    )
-    # flag to indicate whether pyspark dependencies should be included in conda env
-    parser.add_argument(
-        "--add_spark_dependencies",
-        action="store_true",
-        help="include packages for PySpark support",
-    )
-    # path where test logs should be downloaded
-    parser.add_argument(
-        "--testlogs",
-        action="store",
-        default="test_logs.log",
-        help="Test logs will be downloaded to this path",
-    )
-    # conda package name for jdk
     parser.add_argument(
         "--conda_pkg_jdk",
         action="store",
         default="openjdk=8",
-        help="conda package name for jdk",
+        help="Conda package for JDK",
     )
-    # conda package name for python
     parser.add_argument(
-        "--conda_pkg_python",
+        "--python-version",
         action="store",
-        default="python=3.7",
-        help="conda package for Python",
+        default="3.7",
```

Comment:
Probably 3.8, since we don't test 3.7 in the other tests any more.

Reply:
Changed to 3.8